.circleci/test-deploy.yml

```diff
@@ -13,21 +13,45 @@ jobs:
       - cypress/install:
           working-directory: examples/angular-app
           cypress-cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/angular-app/package.json" }}
+          include-branch-in-node-cache-key: true
       - persist_to_workspace:
           root: ~/
           paths:
             - .cache/Cypress
             - project
-  run-tests-in-parallel:
-    executor: cypress/default
-    parallelism: 3
+  run-tests-in-chrome:
+    docker:
+      - image: cypress/browsers:node-16.18.1-chrome-109.0.5414.74-1-ff-109.0-edge-109.0.1518.52-1
+    parallelism: 2
     steps:
       - run: echo "This step assumes dependencies were installed using the cypress/install job"
       - attach_workspace:
           at: ~/
       - cypress/run-tests:
           working-directory: examples/angular-app
-          cypress-command: "npx cypress run --component --parallel --record"
+          cypress-command: "npx cypress run --component --parallel --record --group 2x-chrome --browser chrome"
+  run-tests-in-firefox:
+    docker:
+      - image: cypress/browsers:node-16.18.1-chrome-109.0.5414.74-1-ff-109.0-edge-109.0.1518.52-1
+    parallelism: 2
+    steps:
+      - run: echo "This step assumes dependencies were installed using the cypress/install job"
+      - attach_workspace:
+          at: ~/
+      - cypress/run-tests:
+          working-directory: examples/angular-app
+          cypress-command: "npx cypress run --component --parallel --record --group 2x-firefox --browser firefox"
+  run-tests-in-edge:
+    docker:
+      - image: cypress/browsers:node-16.18.1-chrome-109.0.5414.74-1-ff-109.0-edge-109.0.1518.52-1
+    parallelism: 2
+    steps:
+      - run: echo "This step assumes dependencies were installed using the cypress/install job"
+      - attach_workspace:
+          at: ~/
+      - cypress/run-tests:
+          working-directory: examples/angular-app
+          cypress-command: "npx cypress run --component --parallel --record --group 2x-edge --browser edge"
```

Comment:
We still need to confirm this grouping command works per the docs, yeah?

Reply:
@jennifer-shehane confirmed that this works as expected